src/components/project/monitoring/GraphHeartbeats.js

```diff
@@ -15,21 +15,48 @@ const styles = {
 
 class GraphHeartbeats extends React.Component {
   static getColor(percent) {
-    if (percent > 1) {
-      return '#00E200'
-    }
-    if (percent === 0) {
-      return Theme.colors.neutlight_d2
-    }
-    let r = Math.floor(((((1 - percent) / 0.5) * 228) + 228) / 2)
-    let g = Math.floor((percent / 0.5) * 228)
-    const b = 0
-    if (percent > 0.5) {
-      g = 228
-    } else {
-      r = 228
+    const colors = [
+      {
+        min: 0.9999,
+        color: Theme.colors.green500
+      },
+      {
+        min: 0.99,
+        color: Theme.colors.lightGreen500
+      },
+      {
+        min: 0.95,
+        color: Theme.colors.lime500
+      },
+      {
+        min: 0.9,
+        color: Theme.colors.yellow500
+      },
+      {
+        min: 0.75,
+        color: Theme.colors.amber500
+      },
+      {
+        min: 0.5,
+        color: Theme.colors.orange500
+      },
+      {
+        min: 0.1,
+        color: Theme.colors.deepOrange500
+      },
+      {
+        min: 0.000001,
+        color: Theme.colors.red500
+      }
+    ]
+    let color = Theme.colors.neutlight_d2
+    for (let i = 0; i < colors.length; i += 1) {
+      if (percent > colors[i].min) {
+        ({ color } = colors[i])
```

Comment:
Why do you have parenthesis around this?

Reply:
it made my linter happy...